static/app/views/settings/organizationMembers/organizationMemberRow.tsx

```diff
@@ -224,6 +225,19 @@ export default class OrganizationMemberRow extends PureComponent<Props, State> {
                 {t('Leave')}
               </Button>
             )}
+
+            {showLeaveButton && !memberCanLeave && flags['idp:provisioned'] && (
+              <Button
+                size="sm"
+                icon={<IconClose size="xs" />}
+                disabled
+                title={t(
+                  'You cannot leave this organization as your role has been idp-provisioned.'
```

Comment:
Reuse the "_XYZ_ is managed through your organization's identity provider." template.